app/jobs/tracker_job.rb

```diff
@@ -0,0 +1,32 @@
+class TrackerJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(category, action, label, value, client_id, ga_id)
+    if ga_id.present?
```

Comment:
Partly a result of a refactoring, but I left it in just because there are potential use cases for calling the job directly.. such as using a different ga_id.